### ISSUE TITLE
Update random_state in exported codes

### ISSUE THREAD

**Issue by weixuanfu** [EpistasisLab/tpot#741]
CLI of TPOT allow users to assign different random seed to `train_test_split` rather than None via `-s` parameter (see [this line](https://github.com/EpistasisLab/tpot/blob/v0.9/tpot/driver.py#L485-L486)). But [the `train_test_split` function in exported codes](https://github.com/EpistasisLab/tpot/blob/v0.9/tpot/export_utils.py#L89-L90) is assigned a fixed random seed of 42 as many examples in scikit-learn. So I think it is more clear to use the same `random_state` in exported codes.
